app/views/checkin/new.html.erb

```diff
@@ -1,28 +1,11 @@
-<div class="container">
+<div class="container ">
   <div class="px-4 py-3 my-5 text-center">
     <h2 class="display-5 fw-bold pb-4">Check-in to the Berkeley Reentry <br> Student Program Study Space</h2>
-    <%= form_with(model: @checkin, local: true) do |f| %>
```

Comment:
This form was legacy, so I had to remove the divs and add in a new one for the select dropdown